src/audio/mux/mux_generic.c

```diff
@@ -18,9 +19,9 @@
  * \param[in] offset Offset in source buffer.
  * \param[in] mask Routing bitmask for calculating output sample.
  */
-static inline int32_t calc_sample_s16le(struct comp_buffer *source,
-					uint8_t num_ch, uint32_t offset,
-					uint8_t mask)
+UT_STATIC inline int32_t calc_sample_s16le(struct comp_buffer *source,
```

Comment:
Do we need to pollute the code like this? We could just use the comp API and call this methods implicitly.

Reply:
Defines like this are needed to make it unit-testable. With public API you can test processing as whole (intergration tests), but this is one f.e. is for unit-testing just sample calculation.